src/Php81/Resources/stubs/CURLStringFile.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+if (extension_loaded('curl') && class_exists(CURLFile::class, false)) {
+    // https://php.watch/versions/8.1/CURLStringFile#CURLStringFile-polyfill
+    class CURLStringFile extends CURLFile
+    {
+        private $rawData;
+
+        public function __construct(string $data, string $postname, string $mime = 'application/octet-stream')
+        {
+            $this->rawData = $data;
+            parent::__construct('data://' . $mime . ';base64,' . base64_encode($data), $mime, $postname);
+        }
+
+        public function __set(string $name, $value): void
+        {
+            if ($name !== 'data') {
+                return;
+            }
+
+            if (!is_string($value)) {
```

Comment:
I suppose native PHP also accepts stringable, or integers, etc?
Maybe we should just not do this type check.
Alternatively, the closes we could do is adding the `string` type to the `$value` argument, in the signature.